gBrogue Source/gBrogueCode/IO.c

```diff
@@ -2306,8 +2322,8 @@ void nextBrogueEvent(rogueEvent *returnEvent, boolean textInput, boolean colorsD
 				if (pauseDuration && pauseBrogue(pauseDuration)) {
 					// if the player did something during playback
 					nextBrogueEvent(&recordingInput, false, false, true);
-					executePlaybackInput(&recordingInput);
-					repeatAgain = !rogue.playbackPaused;
+					interaction = executePlaybackInput(&recordingInput);
+					repeatAgain = !rogue.playbackPaused && interaction;
```

Comment:
This is a bugfix to improve the performance of recordings which is required on the web and helps on other platforms.